blocks/chat-cta/chat-cta.js

```diff
@@ -0,0 +1,19 @@
+const init = async (el) => {
+  const cta = el.children[0].children[0];
+  cta.classList.add('cta-chat-sticky');
+
+  if (window.innerWidth <= 900) {
+    const a = cta.querySelector('a');
+    a.classList.add('con-button', 'outline');
+  }
```

Comment:
Since this only happens on init, when resizing `a` doesn't get these classes. Did you consider using a resize listener?
```
window.addEventListener('resize', throttle(
  1000,
  () =>  {
    if (window.innerWidth <= 900) {
        a.classList.add('con-button', 'outline'),
    }
  }
));
```

Reply:
Interesting thought. Maybe we can discuss at the FE Knowledge share.

My gut instinct is that since the experience is only different on mobile, but the same on desktop and tablet, the user is not likely to resize the window when they came to the page. If they did, you are right, the button styles would not get applied, so I think this is perhaps a discussion in what merits a resize event listener.

Reply:
Gonna solve w/ CSS and override.